### PR TITLE
fix: broken JSON

### PR DESCRIPTION
**Pull request by naari3**
I used the workflows provided by tfaction-example as they are, but some errors occurred, so I will fix them.

### CODE REVIEW

File: .github/workflows/wc-plan.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ on:
 jobs:
   plan:
     name: "plan (${{matrix.target.target}})"
-    runs-on: ${{fromJSON(matrix.target.runs-on)}}
+    runs-on: ${{matrix.target.runs_on}}
```
Comment (naari3) [inline .github/workflows/wc-plan.yaml:21]: We believe two issues were at play here.
- `runs-on`  doesn't seem to exist here
- Strings like "ubuntu-latest" don't seem to parse in `fromJSON` maybe??

**Comment by suzuki-shunsuke** [comment]
I think this example is correct.

https://github.com/suzuki-shunsuke/tfaction-example/blob/5a34f16526d5e1fda6a3b2a5fbae9718d63f7021/.github/workflows/wc-test.yaml#L57

`targets` is a JSON string like

```json
[{"runs-on": "\"ubuntu-latest\""}]
```

https://github.com/suzuki-shunsuke/tfaction-example/blob/5a34f16526d5e1fda6a3b2a5fbae9718d63f7021/.github/workflows/wc-plan.yaml#L33

`target` is an object like

```json
{
  "runs-on": "\"ubuntu-latest\""
}
```

https://github.com/suzuki-shunsuke/tfaction-example/blob/5a34f16526d5e1fda6a3b2a5fbae9718d63f7021/.github/workflows/wc-plan.yaml#L21

`runs-on` is an object like

```json
"ubuntu-latest"
```

`runs-on` is either a string such as `ubuntu-latest` or an array such as `["self-hosted", "linux"]`.

Before sending a pull request, could you create an issue?

**Comment by naari3** [comment]
I see, I will create issue and reproducable repositories!

Also, I was trying to solve two problems in the same file at the same time with this PR, but I should have created them separately.
https://github.com/suzuki-shunsuke/tfaction-example/pull/2247/files#diff-50521524435c37bcf28a4bb9cde830214e929844fbda10a8efa6afdfae910d60R69

**Comment by suzuki-shunsuke** [comment]
Oh, I see.
As you said, `runs-on` is a typo of `runs_on`.
Thank you.

**Comment by suzuki-shunsuke** [comment]
Oh, I see.
I misunderstood.
You're correct.

**Pull request merged**